.ruff.toml

```diff
@@ -0,0 +1,43 @@
+target-version = "py310"
+line-length = 110
+exclude = [
+  ".git,",
+  "__pycache__",
+  "build",
+  "ndcube/version.py",
+]
+
+[lint]
+select = ["E", "F", "W", "UP", "PT"]
+extend-ignore = [
+  # pycodestyle (E, W)
+  "E501", # LineTooLong                             # TODO! fix
+  # pytest (PT)
+  "PT001", # Always use pytest.fixture()
+  "PT004", # Fixtures which don't return anything should have leading _
+  "PT007", # Parametrize should be lists of tuples  # TODO! fix
+  "PT011", # Too broad exception assert             # TODO! fix
+  "PT023", # Always use () on pytest decorators
+]
+
+[flake8-tidy-imports]
+[flake8-tidy-imports.banned-api]
+"warnings.warn".msg = "Use sunpy specific warning helpers warn_* from sunpy.utils.exceptions"
+
+[per-file-ignores]
+# Part of configuration, not a package.
+"setup.py" = ["INP001"]
+"conftest.py" = ["INP001"]
+"docs/conf.py" = [
+  "E402"  # Module imports not at top of file
+]
+"docs/*.py" = [
+  "INP001",  # Implicit-namespace-package. The examples are not a package.
+]
+"__init__.py" = ["E402", "F401", "F403"]
+"test_*.py" = ["B011", "D", "E402", "PGH001", "S101"]
+# Need to import clients to register them, but don't use them in file
+"ndcube/net/__init__.py" = ["F811"]
```

Comment:
Is this still in the template we should purge it, it's really sunpy/sunpy specific.